bowling/src/input.cpp

```diff
@@ -0,0 +1,65 @@
+#include <algorithm>
+#include <fstream>
+
+#include "input.hpp"
+
+namespace fs = std::experimental::filesystem;
+
+Input::Input(const std::string& directory) : directoryPath_(fs::path(directory)), isValid_(false) {
+    if (!fs::exists(directoryPath_)) {
+        return;
+    }
+    if (!fs::is_directory(directoryPath_)) {
+        return;
+    }
+    std::vector<fs::path> files = getFiles();
```

Comment:
make files private variable

Reply:
and change `getFiles()`  into `makeFileList()`

Reply:
file list is needed in constructor only, no need keep it private, anyway I like the function name